packages/components/ng-ovh-contracts/src/inline-contracts/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,7 @@
+{
+  "ovh_inline_contracts_label_unique": "Je reconnais avoir bien pris connaissance des {{ contract }}.",
+  "ovh_inline_contracts_label_first": "Je reconnais avoir bien pris connaissance des {{ contract }},",
+  "ovh_inline_contracts_label_next": "et des {{ contract }},",
```

Comment:
I'm not sure it would work for every possible contracts or languages. For example "et des la politique de confidentialité" or stuff like that.

Maybe something like ```"Je reconnais [...] connaissance des contrats : " + contracts.join(", ")``` would be safer?

Reply:
I've told them like 10 times already that this doesn't work 😅 

I suggest a list `<ul><li>...`

Reply:
(also it's a subtree so Marie won't do anything about this)

Reply:
I will make the change on the base component, after this, you can update your PR